### PR TITLE
Initial documentation for ClusterBuster

### PR DESCRIPTION
**Pull request by RobertKrawitz**

### CODE REVIEW

File: docs/clusterbuster.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+### Create A Workload
+
+To create a new workload, you need to do the following:
+
+1. Create a `.workload` file and place it in
+   `lib/clusterbuster/workloads`.  The workload file is a set of bash
+   functions, as the file is sourced by clusterbuster.
+   
+2. (Optional) Create one or more perl scripts, which go into
+   `lib/clusterbuster/pod-files`.  These scripts are written in Perl
```
Comment (gqlo) [inline docs/clusterbuster.md:204]: `pod_files`

File: README.md [hunk 2]
```diff
@@ -61,15 +61,7 @@ etc. OpenShift 4 clusters.
 ## Testing tools
 
 - **clusterbuster** -- generate pods, namespaces, and secrets to stress
-  test a cluster.  Also optionally do client/server data transfer.
-  Documentation is via `-h` or `--help`, `-H` or `--help-all`, or
-  `--help-everything` in increasing degree of completeness.
-
-- **clusterbuster-connstat** -- retrieve information about
-  client/server clusterbuster run.
```
[the remark this answers is not in the document]
Comment (RobertKrawitz) [reply README.md:69]: Yep, this was removed.

File: docs/clusterbuster.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+
+* `<workload>_supports_reporting`
+  Return true (0) if the workload supports reporting, false (non-zero
+  status) otherwise.
+
+* `<workload>_list_configmaps`
+
+  Return a list of files that must be provided to the worker object
+  for the workload to run.  This consists of Perl files in
+  `lib/clusterbuster/pod-files` that are required to run the workload.
```
[the remark this answers is not in the document]
Comment (RobertKrawitz) [reply docs/clusterbuster.md]: Fixed.

**Comment by RobertKrawitz** [comment]
My apologies about missing that you had reviewed this.  I've updated the text.

**Review by gqlo — APPROVED** [review]
Draft looks good.

**Comment by RobertKrawitz** [comment]
OK, I'll merge this, then update it in the Python PR.

**Pull request merged**